metabrainz/templates/admin/users/details.html

```diff
@@ -13,81 +39,160 @@
-    <dl class="dl-horizontal">
-      <dt>ID</dt><dd>{{ user.id }}</dd>
-      <dt>State</dt><dd>{{ user.state }}</dd>
-      <dt>Commercial</dt><dd>{{ user.is_commercial }}</dd>
-      <dt>MusicBrainz ID</dt><dd>{{ user.musicbrainz_id }}</dd>
-      <dt>Creation time</dt><dd>{{ user.created }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Contact name</dt><dd>{{ user.contact_name }}</dd>
-      <dt>Contact email</dt><dd>{{ user.contact_email }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Organization name</dt><dd>{{ user.org_name }}</dd>
-      <dt>Organization website</dt><dd>{{ user.website_url }}</dd>
-      <dt>Organization logo URL</dt><dd>{{ user.org_logo_url }}</dd>
-      <dt>Logo image filename</dt><dd>{{ user.logo_filename }}</dd>
-      <dt>Organization API URL</dt><dd>{{ user.api_url }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Address</dt><dd>{{ user.address_street }}</dd>
-      <dt>City</dt><dd>{{ user.address_city }}</dd>
-      <dt>State</dt><dd>{{ user.address_state }}</dd>
-      <dt>Postal code</dt><dd>{{ user.address_postcode }}</dd>
-      <dt>Country</dt><dd>{{ user.address_country }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Tier</dt><dd>{{ user.tier }}</dd>
-      <dt>Amount pledged</dt><dd>${{ user.amount_pledged }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Good standing</dt><dd>{{ user.good_standing }}</dd>
-      <dt>Featured</dt><dd>{{ user.featured }}</dd>
-    </dl>
-    <dl class="dl-horizontal">
-      <dt>Data usage description</dt><dd>{{ user.data_usage_desc }}</dd>
-      <dt>Organization description</dt><dd>{{ user.org_desc }}</dd>
-    </dl>
   </div>
 
-  {% if user.state == 'pending' or user.state == 'waiting' %}
-    <div>
-      <p>
-        <a href="{{ url_for('usersview.approve', user_id=user.id) }}"
-           class="btn btn-success" role="button">Approve</a>
-        <a href="{{ url_for('usersview.approve', user_id=user.id, limited=True) }}"
-           title="This user is not going to have access to the API."
-           class="btn btn-warning" role="button">Approve (limited)</a>
-        {% if user.state != 'waiting' %}
-          <a href="{{ url_for('usersview.wait', user_id=user.id) }}"
-             class="btn btn-warning" role="button">Put into waiting list</a>
-        {% endif %}
-        <a href="{{ url_for('usersview.reject', user_id=user.id) }}"
-           class="btn btn-danger" role="button">Reject</a>
-      </p>
-      <p class="text-muted">
-        <em>
-          You will be redirected to the next pending user after approving
-          or rejecting this one.
+  <h3>Data access</h3>
+  <div class="row">
+    <div class="form-group col-md-6">
+      <label>State</label>
+      <input class="form-control" type="text" value="{{ user.state }}" readonly>
+      <div style="padding-top: 7px">
+        <em class="text-muted">
+          State determines whether this user has access to the MetaBrainz API.
```

Comment:
"State determines whether this user has access to the MetaBrainz Live Data Feed API."